src/WebJobs.Script.WebHost/WebHooks/DynamicWebHookReceiverConfig.cs

```diff
@@ -20,13 +22,8 @@ public Task<string> GetReceiverConfigAsync(string name, string id)
             // "id" will be the function name
             // we ignore the "name" parameter since we only allow a function
             // to be mapped to a single receiver
-            FunctionSecrets secrets = _secretManager.GetFunctionSecrets(id);
-            if (secrets != null)
-            {
-                return Task.FromResult(secrets.Key);
-            }
-
-            return null;
+            string functionSecret = _secretManager.GetFunctionSecrets(id).Values.FirstOrDefault();
```

Comment:
This is a **TODO** item. We need to construct the `id` in a way that provides additional information (if provided) about the key that should be used. 
Currently, the `id` is just mapped to the function name, but we'll need to introduce a convention so a request can identify the client/key id (either in the URI path, query string, header, etc.) to map to a specific secret